build.sbt

```diff
@@ -42,6 +42,13 @@ lazy val root = project("paradox-material-theme-parent", file("."))
     }
     // #social
     ,
+    // #open-graph
+    Compile / paradoxProperties ++= Map(
+      "project.description" -> "Paradox Material Theme is a theme for Paradox, a static site generator geared towards project documentation",
+      "project.image" -> "https://jonas.github.io/paradox-material-theme/images/material.png"
```

Comment:
Shouldn't this URI be updated to `https://www.scala-sbt.org/sbt-paradox-material-theme/images/material.png`?

Reply:
yep good catch thanks